storage/simulation/cluster.go

```diff
@@ -148,24 +229,18 @@ func (c *Cluster) String() string {
 		buf.WriteString("\n")
 	}
 
-	var storeIDs []int
-	for storeID := range c.stores {
-		storeIDs = append(storeIDs, int(storeID))
-	}
-	sort.Ints(storeIDs)
-
 	buf.WriteString("Store Info:\n")
-	for _, storeID := range storeIDs {
+	for _, storeID := range c.storeIDs {
 		s := c.stores[proto.StoreID(storeID)]
 		buf.WriteString(s.String(storesRangeCounts[proto.StoreID(storeID)]))
 		buf.WriteString("\n")
 	}
 
-	var rangeIDs []int
+	var rangeIDs []proto.RangeID
 	for rangeID := range c.ranges {
-		rangeIDs = append(rangeIDs, int(rangeID))
+		rangeIDs = append(rangeIDs, rangeID)
 	}
-	sort.Ints(rangeIDs)
+	sort.Sort(proto.RangeIDSlice(rangeIDs))
```

Comment:
As mentioned previously, I'd prefer it this way, but I don't mind changing it if you have a very strong opinion on it.